tmt/package_managers/apt.py

```diff
@@ -0,0 +1,93 @@
+from typing import Optional
+
+import tmt.package_managers
+import tmt.utils
+from tmt.package_managers import Installable, Options, escape_packages, provides_package_manager
+from tmt.utils import Command, CommandOutput, ShellScript
+
+
+@provides_package_manager('apt')
+class Apt(tmt.package_managers.PackageManager):
+    probe_command = Command('apt', '--version')
+
+    install_command = Command('install')
+
+    def prepare_command(self) -> tuple[Command, Command]:
+        """ Prepare installation command for rpm-ostree"""
+
+        command = Command()
+        options = Command('-y')
```

Comment:
Better to use `DEBIAN_FRONTEND=noninteractive` instead. Some apt packages have an interactive configuration step even when you `-y`